src/site/stages/build/plugins/add-id-to-subheadings.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Add unique ID to H2s and H3s that aren't in WYSIWYG or accordion buttons
+ */
+
+const cheerio = require('cheerio');
+
+const usedHeaders = [];
+
+let g = 1;
+
+function createUniqueId(headingEl) {
+  const headingString = headingEl.text();
+  const length = 30;
+  let anchor = headingString
+    .trim()
+    .toLowerCase()
+    .replace(/[^\w\- ]+/g, '')
+    .replace(/\s/g, '-')
+    .replace(/-+$/, '')
+    .substring(0, length);
+
+  if (usedHeaders.includes(anchor)) {
+    if (!usedHeaders.includes(`${anchor}-${g}`)) {
+      anchor = `${anchor}-${g}`;
+      g++;
+    }
+  }
+  usedHeaders.push(anchor);
+  return anchor;
+}
+
+function generateHeadingIds() {
+  return (files, metalsmith, done) => {
+    for (const fileName of Object.keys(files)) {
+      const file = files[fileName];
+      let idAdded = false;
+
+      if (fileName.endsWith('html')) {
+        const doc = cheerio.load(file.contents);
+        doc('h2, h3').each((i, el) => {
+          const heading = doc(el);
+          const parent = heading.parent();
+          const isInWysiwyg = parent.hasClass('processed-content');
```

Comment:
Is this only going to check the immediate parent? Is it possible that we might have h elements nested in other elements inside the processed content div?